packages/nestjs-module/src/ogma.provider.ts

```diff
@@ -105,6 +113,26 @@ export function createLoggerProviders(
   ];
 }
 
+export function createRequestScopedLoggerProviders(
+  topic: string | (() => any) | Type<any>,
+): Provider[] {
+  topic = typeof topic === 'function' ? topic.name : topic;
+  const token = createRequestScopedProviderToken(topic);
+  return [
+    {
+      inject: [OGMA_INSTANCE, CONTEXT],
```

Comment:
Why use `CONTEXT` instead of `REQUEST`?

Reply:
Actually there is no difference as `CONTEXT` as you could see, is in fact string `REQUEST`. But I used `CONTEXT` because `REQUEST` sounds for me like it is for http based queries and I didn't want any misunderstandings as that should handle requests for all protocols not just for http (for example on grpc we have `metadata` and `payload` and we get `metadata` from `ExecutionContext` by method `getContext`). But when we decide to remove `microservice` dependency then we will be forced to use `REQUEST` instead of `CONTEXT`

Reply:
What we can do, if you still want to use the word `CONTEXT` to help keep things clearer, is make our own `const CONTEXT = REQUEST` and use that constant. They're all the same anyways. Or just `import { REQUEST as CONTEXT }`